packit_service/models.py

```diff
@@ -761,3 +771,48 @@ def get_by_pipeline_id(cls, pipeline_id: str) -> Optional["TFTTestRunModel"]:
                 .filter_by(pipeline_id=pipeline_id)
                 .first()
             )
+
+
+class InstallationModel(Base):
+    __tablename__ = "github_installations"
+    installation_id = Column(Integer, primary_key=True)
+    account_login = Column(String)
+    account_id = Column(Integer)
+    account_url = Column(String)
+    account_type = Column(String)
+    created_at = Column(DateTime, default=datetime.utcnow)
+    repositories = Column(ARRAY(String))
```

Comment:
It would be awesome to connect here to the GitProject table, but is it an overkill?

Reply:
I was thinking about this too, I can change it